jupyter_book/utils.py

```diff
@@ -123,3 +125,13 @@ def _clean_markdown_cells(ntbk):
                     cell_lines[ii] = line.rstrip('#').rstrip()
             cell.source = '\n'.join(cell_lines)
     return ntbk
+
+
+def read_notebook(path_notebook):
+    """Read in a notebook using either nbformat or jupytext."""
+    suff = op.splitext(path_notebook)[-1]
+    if suff == ".ipynb":
+        ntbk = nbf.read(path_notebook, nbf.NO_CONVERT)
+    elif suff == ".md":
```

Comment:
If you wanted, you could use `jpt.read(path_notebook)` even when the notebook has an `.ipynb` extension.

Reply:
very cool - that's a useful function 👍